src/css/archwizard.scss

```diff
@@ -1,9 +1,97 @@
+// 'background-color-hover', 'symbol-color-hover'.
+//
+// Notes:
+// - The order of wizard step states in this mapping does not affect the order of resulting CSS rules, which is fixed
+//   internally by angular-archwizard.
+// - Depending on the active indicator style, some colors may not be used.  For example, 'background-color-default' and
+//   'background-color-hover' do not make sense in 'large-empty-symbols' indicator style.
+// - You can set a color to `null` to exclude the corresponding property from CSS output.
+//
+// Default colors
+// --------------
+//
+// You can set default colors by using '_' in place of an indicator style or a wizard step state.
+// The `wz-get-color` function looks up the requested color in the following order, picking the first defined entry:
+//
+// - $wz-colors[$indicator-style][$step-state][$color]
+// - $wz-colors[$indicator-style]['_'][$color]
+// - $wz-colors['_'][$step-state][$color]
+// - $wz-colors['_']['_'][$color]
+//
+$wz-colors: (
+  '_': (
+    'default': (
+      'border-color-default': #E6E6E6,
+      'background-color-default': #E6E6E6,
+      'symbol-color-default': #E6E6E6,
+      'border-color-hover': darken(#E6E6E6, 10%),
+      'background-color-hover': darken(#E6E6E6, 5%),
+      'symbol-color-hover': darken(#E6E6E6, 10%),
+    ),
+    'current': (
+      'border-color-default': #808080,
+      'background-color-default': #808080,
+      'symbol-color-default': #808080,
+      'border-color-hover': darken(#808080, 10%),
+      'background-color-hover': darken(#808080, 5%),
+      'symbol-color-hover': darken(#808080, 10%),
+    ),
+    'done': (
+      'border-color-default': #339933,
+      'background-color-default': #339933,
+      'symbol-color-default': #339933,
+      'border-color-hover': darken(#339933, 10%),
+      'background-color-hover': darken(#339933, 5%),
+      'symbol-color-hover': darken(#339933, 10%),
+    ),
+    'optional': (
+      'border-color-default': #38ef38,
+      'background-color-default': #38ef38,
+      'symbol-color-default': #38ef38,
+      'border-color-hover': darken(#38ef38, 10%),
+      'background-color-hover': darken(#38ef38, 5%),
+      'symbol-color-hover': darken(#38ef38, 10%),
+    ),
+    'editing': (
+      'border-color-default': #FF0000,
+      'background-color-default': #FF0000,
+      'symbol-color-default': #FF0000,
+      'border-color-hover': darken(#FF0000, 10%),
+      'background-color-hover': darken(#FF0000, 5%),
+      'symbol-color-hover': darken(#FF0000, 10%),
+    ),
+  ),
+  'large-filled-symbols': (
+    '_': (
+      'symbol-color-default': black,
+      'symbol-color-hover': black,
+    )
+  )
+) !default;
+
+// line color
+$wz-line-color: #E6E6E6 !default;
+
+// label colors
+$wz-label-color-default: #808080 !default;
+$wz-label-color-hover: darken($wz-label-color-default, 20%) !default;
 
 // dot definitions
 $wz-dot-border-width: 2px !default;

```

Comment:
Again we need some place where we define what a `dot` is and what the `dot width` is